client/src/components/DogParkShow.js

```diff
@@ -41,10 +43,40 @@ const DogParkShow = (props) => {
         getPark()
     }, [])
 
+    const deletePark = async () => {
+        try {
+            const response = await fetch(`/api/v1/parks/${parkId}`, { method: "DELETE"})
+            if(!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw error
+            }
+            setShouldRedirect(true)
+        } catch (err) {
+            console.error(`Error in fetch: ${err.message}`)
+        }
+    }
+
+    const handleOnClickDelete = (event) => {
+        event.preventDefault()
+        deletePark()
+    }
+
+    if(shouldRedirect) {
+        return <Redirect push to="/" />
+    } 
+
+    let isAdmin = false
+    if(props.user) {
+        isAdmin = props.user.isAdmin
+    }
+    const message = "Delete this dog park"
+
     return (
         <div className="dog-show-page">
             <div className="grid-y align-left">
                 <h1>{park.name}</h1>
+                <a onClick={handleOnClickDelete} href="#" className="help-text">{isAdmin && message}</a>
```

Comment:
consider a `button` element. an anchor tag `a` should be used when navigating to a different page outside of our application. with a `button` element and/or CSS you can add effects like onHover to make any element look clickable without it being an anchor tag